app.py

```diff
@@ -8,6 +8,7 @@
 # This should not be imported here, but for now we can force Flake8 to ignore the error
 # The "noqa" comment can be deleted when we figure out how to get around this.
 from models.staff_tenant_link import StaffTenantLink  # noqa: F401
+from models.tickets import TicketModel  # noqa: F401
```

Comment:
What do you think about moving these into the `models` directory's `__init__.py` file instead?

Reply:
I don't have a deep understanding of how Flask builds the app so I can't answer this question or even why we would need to import TicketModel in the app.py file now. I think further discussion/investigation may be warranted to see why this is needing to be done in the first place before making any decisions on what to do.

Reply:
@NickSchimek It was a temporary band-aid to get around the circular import problem.  `models/tickets.py` imports `TenantModel`, so `models/tenant.py` can't import `TicketModel`.  But `TicketModel` needed to be imported somewhere in order to create the relationship.

Reply:
ah I see. Thank you. The correct way to fix this is that we should not be importing the `TenantModel` in the `TicketModel`